cypress/e2e/mediashelf/deactivationOfCtlToolsInMediaShelf.feature

```diff
@@ -0,0 +1,129 @@
+        When I click on save external tool button
+        Then I see the tool 'CY Test Tool 2' in external tools table
+        Then I see the tool 'CY Test Tool 2' is deactivated in external tools table
+
+        # Teacher can not see a deactivated tool in media shelf
+        Given I am logged in as a '<teacher>' at '<namespace>'
+        When I go to media shelf
+        Then I see the media shelf page title
+        Then I see the available media line
+        Then I do not see tool 'CY Test Tool 2' in the available media line
+        Then I see 1 tools in the available media line
+        Then I see tool 'CY Test Tool 1' in the available media line
+        When I move tool 'CY Test Tool 1' in to ghost media line
+        Then I see the first media line
+        Then I see tool 'CY Test Tool 1' in the first media line
+
+        # Student can not see a deactivated tool in media shelf
+        Given I am logged in as a '<student>' at '<namespace>'
+        When I go to media shelf
+        Then I see the media shelf page title
+        Then I see the available media line
+        Then I do not see tool 'CY Test Tool 2' in the available media line
+        Then I see 1 tools in the available media line
+        Then I see tool 'CY Test Tool 1' in the available media line
+        When I move tool 'CY Test Tool 1' in to ghost media line
+        Then I see the first media line
+        Then I see tool 'CY Test Tool 1' in the first media line
+
+        # Admin deactivates an existing tool
+        Given I am logged in as a '<admin>' at '<namespace>'
+        When I click on administration in menu
+        When I navigate to new school admin page via sub menu
+        When I click on external tools panel
+        Then I see the external tools table
+        Then I see the tool 'CY Test Tool 1' in external tools table
+        When I click on edit button of tool 'CY Test Tool 1'
+        Then I see the external tools configuration page
+        Then I see the external tool configuration page title
+        Then I see the tool configuration infotext
+        Then I see tool 'CY Test Tool 1' is selected
+        When I deactivate the tool
+        Then I see the deactivate checkbox is 'checked'
+        When I click on save external tool button
+        Then I see the tool 'CY Test Tool 1' in external tools table
+        Then I see the tool 'CY Test Tool 1' is deactivated in external tools table
+
+        # Teacher sees a deactivated tool in media shelf
+        Given I am logged in as a '<teacher>' at '<namespace>'
+        When I go to media shelf
+        Then I see the media shelf page title
+        Then I see tool 'CY Test Tool 1' in the first media line
+        Then I see the deactivated chip on media element 'CY Test Tool 1'
+        # Teacher tries to launch a deactivated tool in media shelf
+        When I try to launch tool 'CY Test Tool 1' in the first media line
+        # Then nothing should happen
```

Comment:
how about: Then I see the media shelf page title ?

Reply:
makes no sense. that has nothing to do with the previous step